mmv1/third_party/terraform/tests/resource_os_config_os_policy_assignment_test.go.erb

```diff
@@ -0,0 +1,147 @@
+<% autogen_exception -%>
```

Comment:
Rather than deleting the samples, there are some knobs you can use. `test_hide` I think: https://github.com/GoogleCloudPlatform/magic-modules/commit/b3f42d7a278b7560a77139d353bf7192c949f814#diff-8ced07677fba860d168f1ed7eb070558938004d2093c7ff0d44826b4444feadbR4-R9

Reply:
The tests fail in CI as well, so they seem rather pointless to keep around if they're too large for examples and don't pass in CI

Reply:
For sure! We'll want at least one though, right?

Reply:
Added back one for test_hide